test/inductor/test_aot_inductor.py

```diff
@@ -1138,6 +1138,70 @@ def forward(self, x, y):
                     exactly=True,
                 ).run(src_code)
 
+    def test_reuse_kernel_dynamic(self):
```

Comment:
@chenyang78 Suggest adding `@skipIfRocm` here to skip these new unit tests for ROCm and keep CI green. We will take a look at these unit tests and try to enable them later.

Reply:
Alternately, you can add an entry `"test_reuse_kernel_dynamic ": fail_cuda(is_skip=True),` here: https://github.com/pytorch/pytorch/blob/d3fcb1717c17a3c3541b67f829c0699e60eb0f3b/test/inductor/test_aot_inductor.py#L2421C13-L2421C67
to skip these unit tests only for "cuda" device_type on ROCm.